app/controllers/documents/document_group_types_controller.rb

```diff
@@ -0,0 +1,59 @@
+class Documents::DocumentGroupTypesController < ApplicationController
+  before_action :authenticate
+  before_action :set_document_group_type, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @document_group_types = DocumentGroupType.all
+  end
+
+  def show
+  end
+
+  def new
+    @document_group_type = DocumentGroupType.new
+  end
+
+  def edit
+  end
+
+  def create
+    @document_group_type = DocumentGroupType.new(document_group_type_params)
+    respond_to do |format|
+      if @document_group_type.save
+        format.html { redirect_to DocumentGroupType, notice: 'Dokumentsamlingstypen skapades!' }            
```

Comment:
Line is too long. [108/100]<br>Unnecessary spacing detected.<br>Trailing whitespace detected.